src/pages/ImportData.js

```diff
@@ -380,6 +390,11 @@ const ImportData = () => {
 
   const importPFB = (namespace, name) => {
     return async () => {
+      // TODO: AJ-1104 will error if there is a policy conflict for protected data
+      if (isProtected(url, 'pfb')) {
```

Comment:
Why do we want to just console log here? Does this code buy us anything? I would say if this is truly to handle security stuff, we should handle this by calling the server, having it handle that, and then catching and erroring based on the 403/404. I am concerned that we could end up with server lists out of sync with this approach. (I am assuming this is going to also be a server check anyway, otherwise this is bypassable by editing the clientside code locally)